testing-basics-garden-project/garden-test002.js

```diff
@@ -0,0 +1,7 @@
+import { Garden } from '../src/Garden';
+
+/* Often, code is tested against assert libraries.
+   Javscript has its own assert funcitons that you can look into.
+   For this project, we're going to use the chai assert library.
+   Require the "chai" module in your code and set it to variable "chai"
+*/
```

Comment:
Again, you have a knack for writing these test descriptions. Hope you don't mind, but I tried to shorten it slightly to fit another quick example in, just on the off chance that someone skipped ahead to this project:

```
/* 
  Often, code is tested against assert libraries.
  JavaScript has its own assert functions that you can look into, but for this project, we're going to use the Chai Assertion Library.
  Require the `"chai"` module in your code and set it to variable `chai`.
  For example, here's how to require Node's File System module:
  `const fs = require("fs");`
*/
```